workers/data_refinery_workers/processors/test_array_express.py

```diff
@@ -165,13 +165,13 @@ def test_success(self, mock_file_management: MagicMock):
 
         # If this file already exists for any reason then we aren't
```

Comment:
You may want to revise these two lines of comments due to the code change. The new code seems to always remove this file if it already exists.

Reply:
Wait but I meant for these lines of comments to be explaining why I'm removing the file if it already exists. Is that unclear? Is there a way I could make that clearer?

Reply:
I think what confused me in the comment is "then we aren't actually testing that it is generated". The code actually removes `output_file` when it already exists. Do you think a comment like this:
```If output_file exists, remove it first.```
would make more sense?

Reply:
Yup, I dig that. Nice and brief :)